src/components/layout/Header/HeaderContacts/HeaderContacts.jsx

```diff
@@ -0,0 +1,83 @@
+/**
+ * HeaderContacts component.
+ * @module components/ItaliaTheme/Header/HeaderContacts/HeaderContacts
+ */
+
+import React from 'react';
+import { UniversalLink } from '@plone/volto/components';
+import { Icon } from 'io-sanita-theme/components';
+import { Container, Row, Col } from 'design-react-kit';
+import './headerContacts.scss';
+
+const HeaderContacts = () => {
+  // exemplo items - arriverà via props
+  const items = [
+    {
+      text_1: 'Prenota servizi e prestazioni',
+      text_2: 'CUP',
+      href: 'https://www.sitoweb.it',
+    },
+    {
+      text_1: 'Richiedi informazioni o fai una segnalazione',
+      text_2: 'URP',
+      href: 'tel:0532123456',
+    },
+    {
+      text_1: 'Prenota servizi e prestazioni',
+      text_2: 'CUP',
+      href: 'mailto:teto@prova.it',
+    },
+  ];
+
+  const getDisplayText = (link) => {
+    if (link.startsWith('tel:')) {
+      return link.replace('tel:', '');
+    } else if (link.startsWith('mailto:')) {
+      return link.replace('mailto:', '');
+    } else {
+      return link;
+    }
+  };
+
+  return (
+    items && (
+      <div className="header-contacts">
+        <Container>
+          <Row>
+            {items.map((item, index) => {
+              return (
+                <Col
+                  md="12"
```

Comment:
e sotto md come si comporta?

Reply:
Se sei d'accordo, farei sempre a 2 colonne. Quando c'è il testo lungo, su tablet il testo fa un casino e viene a capo se ci sono più da 3 colonne. In questo modo uniformiamo sempre a 2 colonne ovunque. Cosa dici?

Desk:
<img width="1473" alt="Schermata 2024-07-31 alle 14 26 18" src="https://github.com/user-attachments/assets/f4b82048-48e6-4e92-9b47-fae9bdf26577">

Tablet:
<img width="964" alt="Schermata 2024-07-31 alle 14 26 25" src="https://github.com/user-attachments/assets/260d9003-6ce0-4a72-88dd-58051c4dd9cc">

Mobile:
<img width="469" alt="Schermata 2024-07-31 alle 14 26 32" src="https://github.com/user-attachments/assets/53f952b3-c56d-40e5-bbc7-711208e48275">